src/libServer/LookupServer.cpp

```diff
@@ -1238,33 +1269,6 @@ Json::Value LookupServer::GetTxBlockByNum(const string& blockNum,
   }
 }
 
-Json::Value LookupServer::GetTxBlockByHash(const string& inputHash,
```

Comment:
This method was previously added but has never been used.

Reply:
👍